crates/flux-refineck/src/type_env.rs

```diff
@@ -329,6 +349,21 @@ impl<'a> TypeEnv<'a> {
         Ok(loc)
     }
 
+    pub(crate) fn unfold_strg_ref(
```

Comment:
Maybe we should make a habit of writing inference rules if they are concise like in this case. I find them useful.

````suggestion
    /// ```
    /// -----------------------------------
    /// Γ ; &strg <ℓ: t> => Γ,ℓ: t ; ptr(ℓ)
    /// ```
    pub(crate) fn unfold_strg_ref(
````